### PR TITLE
Update Linux prerequisites installation guide for MTL iGPU

### PR DESCRIPTION
**Pull request by Oscilloscope98**
## Description

Related issue: https://github.com/analytics-zoo/nano/issues/1701#issuecomment-2431378715

Doc preview:
- https://github.com/Oscilloscope98/ipex-llm/blob/update-mtl-linux-mddocs/docs/mddocs/Quickstart/install_linux_gpu.md
- https://github.com/Oscilloscope98/ipex-llm/blob/update-mtl-linux-mddocs/docs/mddocs/Overview/install_gpu.md#linux

### CODE REVIEW

File: docs/mddocs/Quickstart/install_linux_gpu.md [hunk 1]
```diff
@@ -15,9 +17,144 @@ IPEX-LLM currently supports the Ubuntu 20.04 operating system and later, and sup
...
+```
+If the version displayed is not `6.5.0-35-generic`, downgrade or upgrade the kernel to the recommended version.
+
+#### 2. (Optional) Downgrade / Upgrade to kernel 6.5.0-35
+
+If your current kernel version is not `6.5.0-35-generic`, you could downgrade or upgrade it by:
+
+```bash
+export VERSION="6.5.0-35"
+sudo apt-get install -y linux-headers-$VERSION-generic
```
Comment (qiuxin2012) [inline docs/mddocs/Quickstart/install_linux_gpu.md:40]: Just `sudo apt-get install -y linux-image-$VERSION-generic linux-headers-$VERSION-generic linux-modules-extra-$VERSION-generic`

File: docs/mddocs/Quickstart/install_linux_gpu.md [hunk 2]
```diff
@@ -15,9 +17,144 @@ IPEX-LLM currently supports the Ubuntu 20.04 operating system and later, and sup
...
+```
+
+After rebooting, you can use `uname -r` again to see that your kernel version has been changed to `6.5.0-35-generic`.
+
+#### 3. Enable driver support through `force_probe` flag
+
+Next, you need to enable driver support on kernel `6.5.0-35-generic` through `force_probe` parameter：
+
+```bash
+sudo sed -i "s/GRUB_CMDLINE_LINUX_DEFAULT=\"quiet splash\"/GRUB_CMDLINE_LINUX_DEFAULT=\"quiet splash i915.force_probe=7d55\"/g" /etc/default/grub
```
Comment (qiuxin2012) [inline docs/mddocs/Quickstart/install_linux_gpu.md:63]: We had better get force probe device id from dmesg from:
```
(base) arda@Xiaoxin03-Ubuntu:~$ sudo dmesg  | grep i915
[    3.820895] i915 0000:00:02.0: Your graphics device 7d55 is not properly supported by i915 in this
               kernel version. To force driver probe anyway, use i915.force_probe=7d55
[   64.598709] sof-audio-pci-intel-mtl 0000:00:1f.3: init of i915 and HDMI codec failed
```

Comment (qiuxin2012) [reply docs/mddocs/Quickstart/install_linux_gpu.md:63]: Some MTLs' device IDs are 7d40 and 7d45.

Comment (Oscilloscope98) [reply docs/mddocs/Quickstart/install_linux_gpu.md]: Have updated :)

**Review by qiuxin2012 — APPROVED** [review]

**Comment by Oscilloscope98** [comment]
Merge for now. Will raise further PRs for any comments :)

**Pull request merged**